### ISSUE TITLE
Unknown layer type: SoftmaxCrossEntropyLoss

### ISSUE THREAD

**Issue by youyingyin** [richzhang/colorization#43]
Check failed: registry.count(type) == 1 (0 vs. 1) Unknown layer type: SoftmaxCrossEntropyLoss (known types: AbsVal, Accuracy, ArgMax, BNLL, BatchNorm, BatchReindex, Bias, Concat, ContrastiveLoss, Convolution, Crop, Data, Deconvolution, Dropout, DummyData, ELU, Eltwise, Embed, EuclideanLoss, Exp, Filter, Flatten, HDF5Data, HDF5Output, HingeLoss, Im2col, ImageData, InfogainLoss, InnerProduct, Input, LRN, LSTM, LSTMUnit, Log, MVN, MemoryData, MultinomialLogisticLoss, PReLU, Parameter, Pooling, Power, Python, RNN, ReLU, Reduction, Reshape, SPP, Scale, Sigmoid, SigmoidCrossEntropyLoss, Silence, Slice, Softmax, SoftmaxWithLoss, Split, TanH, Threshold, Tile, WindowData)

I have already write the below code in ~/.bashrc, also reboot
export PYTHONPATH=/home/my/Documents/my/colorization/resources/:$PYTHONPATH

**Comment by youyingyin** [comment]
that is because i did not recompile the given caffe version

**Comment by beautifulSun** [comment]
I have the same problem as you. Could you tell me which version of caffe you compiled?
